api4/job.go

```diff
@@ -35,6 +37,37 @@ func getJob(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	forceDownload, _ := strconv.ParseBool(r.URL.Query().Get("download"))
+	if forceDownload {
+		fileName := "actiance_export.xml"
+		fileMime := "application/xml"
+		if job.Data["export_type"] == "csv" {
+			fileName = "csv_export.zip"
+			fileMime = "applicaiton/zip"
+		}
+
+		filePath := "export/" + job.Id + "/" + fileName
+		data, readFileErr := c.App.ReadFile(filePath)
```

Comment:
Appears this is only being read to get the size of the file. Does it make more to write a function to just retrieve the filesize? I'm not sure how large these files get, but I would guess they are at least a few megabytes. And if it's coming from AWS, it would be retrieved twice.

Reply:
**SELF NOTE** Make this another endpoint `/download`

Reply:
Done!